test/cli/template-test-run/remote-fail/expected-stderr

```diff
@@ -1,4 +1,4 @@
 FAIL keboola/my-template/0.0.1 one remote
 Error:
 - Running remote test "one" for template "my-template" failed:
-  - Job "%s" failed: [%s] logger.ERROR: The 'config' section is required in the configuration. [] []
+  - Job "%s" failed: Internal Server Error occurred.
```

Comment:
Ta testovací komponenta nemá ošetřený user errory.
![CleanShot 2023-01-06 at 21 47 48@2x](https://user-images.githubusercontent.com/215660/211097348-568bf94f-8f2c-4108-afc3-b70c05768b71.jpg)

Reply:
To by som asi nahlasil Odinovi, neviem ci chceme generovat `Internal Server Error occurred.` do logov, to by mohlo byt este horsie ako je to teraz.

Reply:
Jo jo, taky se mi to moc nezdá.